nsdb-cluster/src/test/scala/io/radicalbit/nsdb/cluster/coordinator/MetadataCoordinatorSpec.scala

```diff
@@ -49,7 +49,7 @@ class MetadataCoordinatorSpec
         ConfigFactory
           .load()
           .withValue("akka.actor.provider", ConfigValueFactory.fromAnyRef("cluster"))
-          .withValue("akka.remote.netty.tcp.port", ConfigValueFactory.fromAnyRef(2652))
+          .withValue("akka.remote.artery.canonical.port", ConfigValueFactory.fromAnyRef(2650))
```

Comment:
Just for curiosity, why have you changed the port?

Reply:
Honestly, I think it's a refuse most likely a lack of lucidity given by the excess of food that I had during the holidays :)
Anyway the point here is that the akka cluster port must be different than the default one, which is 2552. So this change does not break anything and we can leave it if we all agree